AirCasting/SessionViews/MeasurementDotView.swift

```diff
@@ -2,11 +2,14 @@
 //
 
 import SwiftUI
+import Resolver
 
 struct MeasurementDotView: View {
     
     let value: Double
     @ObservedObject var thresholds: SensorThreshold
+    @InjectedObject private var userSettings: UserSettings
+    @ObservedObject var stream: MeasurementStreamEntity
```

Comment:
let's keep simple views simple and come up with better solutions than adding more dependencies and logic in here